docs/ops/sort/NonMaxSuppression_9.md

```diff
@@ -0,0 +1,124 @@
+# NonMaxSuppression {#openvino_docs_ops_sort_NonMaxSuppression_9}
+
+**Versioned name**: *NonMaxSuppression-9*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: *NonMaxSuppression* performs non maximum suppression of the boxes with predicted scores.
+
+**Detailed description**: *NonMaxSuppression* performs non maximum suppression algorithm as described below:
+
+1.  Let `B = [b_0,...,b_n]` be the list of initial detection boxes, `S = [s_0,...,s_N]` be  the list of corresponding scores.
+2.  Let `D = []` be an initial collection of resulting boxes.
+3.  If `B` is empty then go to step 8.
+4.  Take the box with highest score. Suppose that it is the box `b` with the score `s`.
+5.  Delete `b` from `B`.
+6.  If the score `s` is greater or equal than `score_threshold`  then add `b` to `D` else go to step 8.
+7.  For each input box `b_i` from `B` and the corresponding score `s_i`, set `s_i = s_i * func(IOU(b_i, b))` and go to step 3.
+8.  Return `D`, a collection of the corresponding scores `S`, and the number of elements in `D`.
+
+when `soft_nms_sigma == 0`, `func(iou) = 1 if iou <= iou_threshold else 0`, else `func(iou) = exp(-0.5 * iou * iou / soft_nms_sigma)`.
```

Comment:
incomplete sentence. Please correct it as:
```
Here `func(iou) = 1 if iou <= iou_threshold else 0` when `soft_nms_sigma == 0`, else `func(iou) = exp(-0.5 * iou * iou / soft_nms_sigma)`.
```

Reply:
@rkazants fixed, thanks!